hooks/009-fix-console-conf.chroot

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -eu
+
+rm /usr/lib/systemd/system/getty@.service.d/console-conf.conf
+
+cat <<EOF >/usr/lib/systemd/system/getty@.service.d/console-conf.conf
+[Unit]
+ConditionPathExists=/var/lib/console-conf/complete
+ConditionPathExists=!/run/snapd-recovery-chooser-triggered
+EOF
+
+rm /usr/lib/systemd/system/serial-getty@.service.d/console-conf-serial.conf
+
+cat <<EOF >/usr/lib/systemd/system/serial-getty@.service.d/console-conf-serial.conf
+[Unit]
+ConditionPathExists=/var/lib/console-conf/complete
```

Comment:
Why not check also for snapd-recovery-chooser-triggered here?

Reply:
Good point!

Reply:
`serial-console-conf@.service` only has `ConditionPathExists=!/var/lib/console-conf/complete`. And we want the complement of it. So it is only `ConditionPathExists=/var/lib/console-conf/complete` we want.